### ISSUE TITLE
Can't run on windows 10 (thread main panicked)

### ISSUE THREAD

**Issue by jeffhaskin** [DouglasDwyer/octo-release#12]
I ran and the command prompt showed briefly, then disappeared. So I ran it from command prompt and this is what was returned:

```
Starting persistent threadpool with 5 worker(s).
thread 'main' panicked at C:\Users\Douglas\.cargo\registry\src\index.crates.io-6f17d22bba15001f\hashbrown-0.14.3\src\raw\mod.rs:86:40:
Hash table capacity overflow
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

**Comment by DouglasDwyer** [comment]
Huh, that's very interesting. Thank you for the bug report. Can you provide some additional details about your setup for me?

- Does this happen every single time?
- It sounds like the game crashes even before getting to the main menu. Is that correct?
- Can you run the game from the web browser successfully?
- What are your system specs (CPU model, GPU model, and RAM?)

**Comment by jeffhaskin** [comment]
Apologies for delay. Just downloaded the most recent version, still happening. Here's the additional info:

- It happens every time.
- Correct, no menu appears. shell/command prompt window appears briefly, had to run from command prompt to see the message and run with additional info.
- Online demo doesn't open. Blank white screen.

System Specs:
- Intel(R) Core(TM) i7-3632QM CPU @ 2.20GHz   2.20 GHz
- 16.0 GB (15.9 GB usable) Ram (1600mhz)
- NVIDIA GeForce GT 630M
- Windows 10 Home 22H2 (os build: 19045.4529)
- Windows Feature Experience Pack 1000.19058.1000.0

Browser: Edge

**Comment by DouglasDwyer** [comment]
I see. I'm not sure why the hashmap issue is occurring (the binary should be compatible with your CPU), but I can say for certain that your GPU is unsupported. The native demo requires a Vulkan-compatible device, but [the NVIDIA GeForce GT 630M does not have Vulkan](https://en.wikipedia.org/wiki/GeForce_600_series).

As the hardware is below minimum specifications, I am not planning to debug this issue further. However, I definitely want to add an "unsupported hardware" message or something more user-friendly than a crash. I will leave the ticket open until then.

Thanks for your interest in the demo, and I really appreciate you following up! Let me know if you have any other questions or suggestions.

**Comment by jeffhaskin** [comment]
Makes sense. I'm planning a full factory reset soon anyway, so I'll do some cpu-only testing again after that.
